### DISCUSSION TITLE
HPE Proliant Microserver Gen 10 Doesn't Support SO?

### DISCUSSION THREAD

**Discussion by kiot389** [Security-Onion-Solutions/securityonion#5645] [Unsupported Versions] [answered]
I bought a new HPE Proliant Microserver Gen 10 Xeon E-2224. Security Onion doesn't install. I get the splash screen followed by issues in the stdout. This is an install right out of the box, so I don't think its supported. The specs look good to me. Ubuntu Server & RHEL installed in just a few minutes with no issues. Not sure how the hardware isn't running the SO install. 
https://www.newegg.com/HPE-ProLiant-MicroServer-Gen10-Plus-P16006-001/p/09Z-01S1-000N5

**Comment by clairmont32** [comment]
What version of Ubuntu? Currently only CentOS 7 and Ubuntu 18.04 are supported but 20.04 is coming soon based on a PR.

**Comment by n0th1nt0lOOz** [comment]
You use VMware or proxmox for VM Manager? I have Gen9 and Gen10 and work well with SO

**Comment by kiot389** [comment]
CentOS 7 from the SO image. The system is good. I  have Arkime running on Ubuntu server. Before this HPE, I installed on a DELL workstation, but so-setup would error. The previous version of SO always worked for me. I set it up in a few environments and labs. I can't tell what's happening here. Burning the image was tricky too.

**Comment by dougburks** [comment, marked as answer]
If you haven't already, I'd suggest burning the image using Balena Etcher as described here:
https://docs.securityonion.net/en/2.3/download.html

If that doesn't help, then check the following:
https://docs.securityonion.net/en/2.3/trouble-booting.html

If you still can't get our ISO image to work, then you can always install standard CentOS 7 or Ubuntu 18.04 and then add our Security Onion components manually:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos
